files/fr/web/html/link_types/preload/index.html

```diff
@@ -0,0 +1,222 @@
+
+<h2 id="scripting_and_preloads">Scripts et préchargement</h2>
+
+<p>Une autre fonctionnalité notable est la possibilité d'exécuter le préchargement à l'aide d'un script. Par exemple, voici la création d'une instance {{domxref("HTMLLinkElement")}} qui est ensuite attachée au <i lang="en">DOM</i> :</p>
+
+<pre class="brush: js">var preloadLink = document.createElement(&quot;link&quot;);
+preloadLink.href = &quot;myscript.js&quot;;
+preloadLink.rel = &quot;preload&quot;;
+preloadLink.as = &quot;script&quot;;
+document.head.appendChild(preloadLink);
+</pre>
+
+<p>Cela signifie que le navigateur va précharger le fichier <code>myscript.js</code>, mais ne va pas réellement l'utiliser directement. Pour l'utiliser, vous pouvez faire ceci :</p>
+
+<pre class="brush: js">var preloadedScript = document.createElement(&quot;script&quot;);
+preloadedScript.src = &quot;myscript.js&quot;;
+document.body.appendChild(preloadedScript);
+</pre>
+
+<p>C'est utile lorsque vous voulez précharger un script mais repousser son exécution au moment exact où vous en avez réellement besoin.</p>
+
+<h2 id="other_resource_preloading_mechanisms">Autres ressources concernant les mécanismes de préchargement</h2>
+
+<p>D'autres fonctionnalités de préchargement sont disponibles, mais sachez qu'aucune d'entre elle n'est aussi adaptée à l'objectif recherché que <code>&lt;link rel="preload"&gt;</code>. En voici la liste :</p>
+
+<ul>
+  <li><code>&lt;link rel="prefetch"&gt;</code> est pris en charge depuis longtemps par les navigateurs, mais est destinée à la pré-analyse des ressources qui vont être utilisées lors de la <strong><em>prochaine</em></strong> page visitée/chargée (par exempel lorsque vous vous rendrez sur une autre page du site). C'est une bonne chose, mais ce n'est pas utile pour la page en cours de chargement ! Par ailleurs, les navigateurs donnent une priorité plus faible à <code>prefetch</code> qu'à <code>preload</code> — la page courante est prioritaire par rapport à la page suivante. Consultez la <a href="/fr/docs/Web/HTTP/Link_prefetching_FAQ">FAQ sur le préchargement des liens</a> pour plus de détails ;</li>
+  <li><code>&lt;link rel="prerender"&gt;</code> effectue le rendu une page spécifiée en arrière-plan, ce qui permet d'accèlerer son chargement si la personne vistant le site se rend sur cette page par la suite. Du fait du gaspillage potentiel de bande passante pour les visiteurs (on charge des éléments sans savoir si on en aura besoin), Chrome traite l'instruction <code>prerender</code> comme une instruction <a href="https://developers.google.com/web/updates/2018/07/nostate-prefetch">NoState prefetch</a> ;</li>
```

Comment:
accèlerer > accélérer 
vistant > visitant

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1808/commits/da7e3ddba9e4a7175a9aee11ed0f9d06de6f6ad8